static/app/components/modals/dashboardWidgetLibraryModal/index.tsx

```diff
@@ -98,6 +99,19 @@ function DashboardWidgetLibraryModal({
                   setErrored(true);
                   return;
                 }
+                trackAdvancedAnalyticsEvent('dashboards_views.widget_library.add', {
+                  organization,
+                  num_widgets: selectedWidgets.length,
+                });
+                selectedWidgets.map(selectedWidget => {
```

Comment:
```suggestion
                selectedWidgets.forEach(selectedWidget => {
```

We can use `forEach` since we're not creating an output array